queries/fusion/highlights.scm

```diff
@@ -1,7 +1,24 @@
 (comment) @comment
+(afx_comment) @comment
+
+; identifiers afx
+(afx_opening_element
+  (afx_identifier) @tag)
+(afx_closing_element
+  (afx_identifier) @tag)
+(afx_element_self_closing
+  (afx_identifier) @tag)
+
+(afx_attribute
+  (afx_property_identifier) @attribute)
```

Comment:
This is probably `@tag.attribute` https://github.com/theHamsta/nvim-treesitter/blob/1111827b36324bc4e80b6d493e61a85aeca592ef/CONTRIBUTING.md#L187
`@attribute` is more like C++-attribute (an annotation).